CHANGELOG.md

```diff
@@ -27,6 +27,8 @@
       - libOSRM now creates an own watcher thread then used in shared memory mode to listen for data updates
     - Tools:
       - Added osrm-extract-conditionals tool for checking conditional values in OSM data
+    - Trip Plugin
+      - Added a new feature that finds the optimal route given a list of location, a start and destination locations. This does not return a roundtrip, but instead gives the optimal route starting at the start and destination locations.
```

Comment:
"... optimal route starting at the start and destination locations" - > "... optimal route given fixed start and end points"